velox/docs/functions/spark/string.rst

```diff
@@ -23,6 +23,21 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT contains('Spark SQL', null); -- NULL
         SELECT contains(x'537061726b2053514c', x'537061726b'); -- true
 
+.. spark:function:: conv(number, fromBase, toBase) -> varchar
+
+    Converts ``number`` represented as a string from ``fromBase`` to ``toBase``.
+    ``fromBase`` must be a value inclusively between 2 and 36. ``toBase`` must be a value
```

Comment:
typos:

``fromBase`` is an INTEGER value between 2 and 36 inclusively.

``toBase`` is an INTEGER value between 2 and 36 inclusively or between -36 and -2 inclusively.